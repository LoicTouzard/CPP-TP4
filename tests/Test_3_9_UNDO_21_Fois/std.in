C c1 0 0 1
C c2 0 0 2
C c3 0 0 3
C c4 0 0 4
C c5 0 0 5
C c6 0 0 6
C c7 0 0 7
C c8 0 0 8
C c9 0 0 9
C c10 0 0 10
C c11 0 0 11
C c12 0 0 12
C c13 0 0 13
C c14 0 0 14
C c15 0 0 15
C c16 0 0 16
C c17 0 0 17
C c18 0 0 18
C c19 0 0 19
C c20 0 0 20
C c21 0 0 21
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
EXIT
