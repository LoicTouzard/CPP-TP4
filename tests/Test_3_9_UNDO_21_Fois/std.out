OK
#New object :c1
OK
#New object :c2
OK
#New object :c3
OK
#New object :c4
OK
#New object :c5
OK
#New object :c6
OK
#New object :c7
OK
#New object :c8
OK
#New object :c9
OK
#New object :c10
OK
#New object :c11
OK
#New object :c12
OK
#New object :c13
OK
#New object :c14
OK
#New object :c15
OK
#New object :c16
OK
#New object :c17
OK
#New object :c18
OK
#New object :c19
OK
#New object :c20
OK
#New object :c21
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
#nothing to undo !
